client/src/components/common/SearchBar/index.jsx

```diff
@@ -0,0 +1,23 @@
+import React, { Fragment } from 'react';
+import './style.css';
+import PropTypes from 'prop-types';
+import { FontAwesomeIcon } from '@fortawesome/react-fontawesome';
+import { faSearch } from '@fortawesome/free-solid-svg-icons';
+
+const SearchBar = (props) => {
+  const { className } = props;
```

Comment:
The submit button in this component will refresh the page, and we probably wouldn't have any control over it (we would need to prevent default on the submit so it wouldn't refresh, and then we will probably handle the data it deals with in the click handler function.

I suggest you pass a ```handleSubmit``` function from props. This way you can define later it in the class that will render this component and pass it as props.